src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -295,6 +323,18 @@ def execution_environment_volume_mounts(
                     return messages, exit_messages
 
             entry.value.current = parsed_volume_mounts
+
+        if self.extra_volume_mounts and entry.value.current is C.NOT_SET:
+            entry.value.current = []
+
+        for mount in self.extra_volume_mounts:
+            if not mount.exists():
+                exit_msg = (
+                    f"The volume mount source path '{mount.src}', needed by "
+                    "a configuration option, does not exist."
+                )
```

Comment:
(commentary) oh....... it this gets hit it means a different post processor added an invalid volume mount, yuck.

If the dataclass was extended to include another attribute called "source_post_processor", we could use it here so when the bug gets filed we might be able to look at the specific one.